code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -104,15 +104,27 @@
 
 		if("howl", "howls")
 			var/M = handle_emote_param(param) //Check to see if the param is valid (mob with the param name is in view).
-			message = "<B>[src]</B> howls[M ? " at [M]" : ""]!"
-			playsound(loc, 'sound/goonstation/voice/howl.ogg', 100, 0, 10)
-			m_type = 2
+			if(miming)
+				message = "<B>[src]</B> sticks \his head back apparently braying at some moon."
```

Comment:
may need to be re-worded....wording is a bit awkward...mostly because of the "braying", since that's....typically associated with mules and donkies.